### PR TITLE
Update ScottoAlp handwired keyboard to 12 column layout

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description

Added extra pins and keys for a 12-column layout. Because this board is handwired, I am just using `LAYOUT` and the extra columns can be left off or added depending on the variant.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Keyboard updates must go through `develop`.

**Comment by joe-scotto** [comment]
@fauxpark I changed the target branch to `develop`, is that all that needs to change?

File: keyboards/scottokeebs/scotto34/info.json [hunk 1]
```diff
@@ -14,22 +14,48 @@
...
+            "GP10",
+            "GP11",
+            "GP12",
+            "GP13",
+            "GP18",
+            "GP19",
+            "GP20",
+            "GP21",
+            "GP22"
+        ],
```
Comment (fauxpark) [inline keyboards/scottokeebs/scotto34/info.json:28]:
```suggestion
        "cols": ["GP9", "GP10", "GP11", "GP12", "GP13", "GP18", "GP19", "GP20", "GP21", "GP22"],
```

File: keyboards/scottokeebs/scotto34/info.json [hunk 2]
```diff
@@ -14,22 +14,48 @@
...
+        },
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
```
Comment (fauxpark) [inline keyboards/scottokeebs/scotto34/info.json:55]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**